src/main/java/io/github/pmckeown/dependencytrack/upload/UploadBomMojo.java

```diff
@@ -63,6 +77,15 @@ public void performAction() throws MojoExecutionException, MojoFailureException
                 handleFailure("Bom upload failed");
             }
             Project project = projectAction.getProject(projectName, projectVersion);
+            if (updateProjectInfo) {
+                Optional<ProjectInfo> info = createProjectInfo();
```

Comment:
Why not use the Cyclone DX Java APIs for parsing and pulling the Project Info?  Would be better than manually parsing and would allow easier handling of new versions of the CycloneDX Spec.

Reply:
https://github.com/CycloneDX/cyclonedx-core-java